### PR TITLE
[Master] add  interfaces to get guests list from the host

### PR DESCRIPTION
**Pull request by trenxiaof**
Add host_get_guest_list interfaces to get all the guests from hypervisor, which is needed by CIC onboarding feature.

### CODE REVIEW

**Comment by trenxiaof** [comment]
@jichenjc @bjhuangr @SeanHQF please help review

File: doc/source/restapi.rst [hunk 1]
```diff
@@ -1260,7 +1260,33 @@ Delete Guest nic
 Host
 ====
 
-Get info from host (hypervisor) running on.
```
Comment (SeanHQF) [inline doc/source/restapi.rst]: I think you wrongly removed this line which belongs to 'Get Host Info'. Pls take it back.

**Comment by SeanHQF** [comment]
code generally lgtm, pls attach the result file of 'tox -v' @trenxiaof

File: doc/source/restapi.rst (hunk 1 above)
Comment (trenxiaof) [reply doc/source/restapi.rst]: yes, will restore. Thank you!

**Comment by trenxiaof** [comment]
> code generally lgtm, pls attach the result file of 'tox -v' @trenxiaof

OK.
[tox_v_host_get_guest_list.txt](https://github.com/openmainframeproject/python-zvm-sdk/files/4274558/tox_v_host_get_guest_list.txt)

**Comment by SeanHQF** [comment]
> > code generally lgtm, pls attach the result file of 'tox -v' @trenxiaof
> 
> OK.
> [tox_v_host_get_guest_list.txt](https://github.com/openmainframeproject/python-zvm-sdk/files/4274558/tox_v_host_get_guest_list.txt)

Rebase your code on the latest and solve the pep8/doc errors:
ERROR: InvocationError for command /root/repo/python-zvm-sdk/.tox/docs/bin/sphinx-build -W -b html -d doc/build/doctrees doc/source doc/build/html (exited with code 2)
___________________________________ summary ____________________________________
ERROR:   pep8: commands failed
  py27: commands succeeded
  py36: commands succeeded
ERROR:   docs: commands failed

**Comment by trenxiaof** [comment]
> > > code generally lgtm, pls attach the result file of 'tox -v' @trenxiaof
> > 
> > 
> > OK.
> > [tox_v_host_get_guest_list.txt](https://github.com/openmainframeproject/python-zvm-sdk/files/4274558/tox_v_host_get_guest_list.txt)
> 
> Rebase your code on the latest and solve the pep8/doc errors:
> ERROR: InvocationError for command /root/repo/python-zvm-sdk/.tox/docs/bin/sphinx-build -W -b html -d doc/build/doctrees doc/source doc/build/html (exited with code 2)
> ___________________________________ summary ____________________________________
> ERROR: pep8: commands failed
> py27: commands succeeded
> py36: commands succeeded
> ERROR: docs: commands failed

yes, the docs error has been in the latest master. will rebase it

**Comment by SeanHQF** [comment]
Like what I said, if the errors exist in latest master branch, open issue for it. And try to check the result  of your ut cases and docs e.g. by skipping the failed cases.

File: doc/source/restapi.rst (hunk 1 above)
Comment (SeanHQF) [reply doc/source/restapi.rst]: yw

**Comment by trenxiaof** [comment]
ERROR:   pep8: commands failed  
  py27: commands succeeded
  py36: commands succeeded
  docs: commands succeeded

pep8 failed:
[9286] /root/repo/python-zvm-sdk$ /root/repo/python-zvm-sdk/.tox/pep8/bin/flake8
./get-pip.py:90:5: E306 expected 1 blank line before a nested definition, found 0
./zvmsdk/tests/unit/test_volumeop.py:584:80: E501 line too long (83 > 79 characters)
./zvmsdk/tests/unit/test_volumeop.py:589:80: E501 line too long (83 > 79 characters)
./zvmsdk/tests/unit/test_volumeop.py:800:80: E501 line too long (84 > 79 characters)
./zvmsdk/tests/unit/test_volumeop.py:818:80: E501 line too long (83 > 79 characters)
./zvmsdk/tests/unit/test_volumeop.py:823:80: E501 line too long (83 > 79 characters)
./zvmsdk/tests/fvt/test_volume.py:160:80: E501 line too long (80 > 79 characters)
it's not related this commit.

**Comment by SeanHQF** [comment]
code lgtm, @jichenjc @bjhuangr pls help to merge if no other comments. thanks

**Comment by SeanHQF** [comment]
> ERROR: pep8: commands failed
> py27: commands succeeded
> py36: commands succeeded
> docs: commands succeeded
> 
> pep8 failed:
> [9286] /root/repo/python-zvm-sdk$ /root/repo/python-zvm-sdk/.tox/pep8/bin/flake8
> ./get-pip.py:90:5: E306 expected 1 blank line before a nested definition, found 0
> ./zvmsdk/tests/unit/test_volumeop.py:584:80: E501 line too long (83 > 79 characters)
> ./zvmsdk/tests/unit/test_volumeop.py:589:80: E501 line too long (83 > 79 characters)
> ./zvmsdk/tests/unit/test_volumeop.py:800:80: E501 line too long (84 > 79 characters)
> ./zvmsdk/tests/unit/test_volumeop.py:818:80: E501 line too long (83 > 79 characters)
> ./zvmsdk/tests/unit/test_volumeop.py:823:80: E501 line too long (83 > 79 characters)
> ./zvmsdk/tests/fvt/test_volume.py:160:80: E501 line too long (80 > 79 characters)
> it's not related this commit.

Ok, open issue for it and ask related folk to solve. thanks

**Comment by trenxiaof** [comment]
> > ERROR: pep8: commands failed
> > py27: commands succeeded
> > py36: commands succeeded
> > docs: commands succeeded
> > pep8 failed:
> > [9286] /root/repo/python-zvm-sdk$ /root/repo/python-zvm-sdk/.tox/pep8/bin/flake8
> > ./get-pip.py:90:5: E306 expected 1 blank line before a nested definition, found 0
> > ./zvmsdk/tests/unit/test_volumeop.py:584:80: E501 line too long (83 > 79 characters)
> > ./zvmsdk/tests/unit/test_volumeop.py:589:80: E501 line too long (83 > 79 characters)
> > ./zvmsdk/tests/unit/test_volumeop.py:800:80: E501 line too long (84 > 79 characters)
> > ./zvmsdk/tests/unit/test_volumeop.py:818:80: E501 line too long (83 > 79 characters)
> > ./zvmsdk/tests/unit/test_volumeop.py:823:80: E501 line too long (83 > 79 characters)
> > ./zvmsdk/tests/fvt/test_volume.py:160:80: E501 line too long (80 > 79 characters)
> > it's not related this commit.
> 
> Ok, open issue for it and ask related folk to solve. thanks

have opened issue to benlei

**Pull request merged**